mobile/src/main.jsx

```diff
@@ -54,6 +55,9 @@ const renderAppWithPersistedState = persistedState => {
     if (navigator && navigator.splashscreen) navigator.splashscreen.hide()
   }, false)
 
+  useHistoryForTracker(hashHistory)
+  if (store.getState().mobile.settings.analytics) startTracker(store.getState().mobile.settings.serverUrl)
```

Comment:
@kosssi : this is the part I'm the least sure about. Is this the right place to boot the tracking code, or should it be in the `deviceReady` callback or something like that?